libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -80,17 +93,49 @@ export class RadioGroupComponent extends CollectionBaseInput implements AfterVie
         this.id = `radio-group-${nextUniqueId++}`;
     }
 
+    /**
+     * controlvalue accessor
+     */
+    writeValue(value: any): void {
+        if (value) {
+            this._value = value;
+            this.onChange(value);
+        }
+    }
+
+    /**
+     * acess display value for objects, acts as checkbox label.
```

Comment:
same as above

Reply:
Done this and other places where it was required.